src/App.tsx

```diff
@@ -33,6 +34,7 @@ const App = () => {
             {/* TODO: Eventually we will need to differentiate between game page and 
             custome game page */}
             <Route path="/games/:id" element={<GamePageWrapper/>}></Route>
+            <Route path="/games/instances/:id" element={<GameInstancePageWrapper/>}></Route>
```

Comment:
If we go ahead with the above we wouldn't need `instances` in the url which might be nice to avoid (could be confusing for users?)